Generals/Code/GameEngine/Source/GameClient/InGameUI.cpp

```diff
@@ -5484,6 +5479,24 @@ void InGameUI::selectNextIdleWorker( void )
 	}
 }
 
+// Finds unique selectables to avoid selecting the same or a previous container if multiple idle workers are contained
+std::vector<Object*> InGameUI::getUniqueIdleWorkers(const ObjectList& idleWorkers)
```

Comment:
You created a `ObjectPtrVector` but are not using it much.

Reply:
Fixed!